<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1307</width>
    <height>836</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>8</number>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <column>
       <property name="text">
        <string>نام</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>نام خانوادگی</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>نام پدر</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>شماره شناسنامه</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>تصویر </string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetlayout" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="bottomMargin">
        <number>50</number>
       </property>
       <item>
        <widget class="QPushButton" name="show_all">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>70</height>
          </size>
         </property>
         <property name="text">
          <string>نمایش همه ی رکورد ها</string>
         </property>
         <property name="icon">
          <iconset resource="../booking/booking.qrc">
           <normaloff>:/choose_patient/show_all.png</normaloff>:/choose_patient/show_all.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="shortcut">
          <string>Ctrl+A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearField">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>70</height>
          </size>
         </property>
         <property name="text">
          <string>پاک کردن  همه فیلد ها</string>
         </property>
         <property name="icon">
          <iconset resource="../booking/booking.qrc">
           <normaloff>:/choose_patient/clear_fields.png</normaloff>:/choose_patient/clear_fields.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>45</width>
           <height>45</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="first_name">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>نام</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="last_name">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>نام خانوادگی</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="father_name">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>نام پدر</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pass_id">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>شماره شناسنامه</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="search">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>70</height>
          </size>
         </property>
         <property name="text">
          <string>جستجو         </string>
         </property>
         <property name="icon">
          <iconset resource="../booking/booking.qrc">
           <normaloff>:/choose_patient/search.png</normaloff>:/choose_patient/search.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="shortcut">
          <string>Ctrl+F</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>مشخصات بیمار انتخاب شده</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="first_name_s">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>نام</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="last_name_s">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>نام خانوادگی</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="father_name_s">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>نام پدر</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pass_id_s">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>شماره شناسنامه</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="select_person">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>70</height>
          </size>
         </property>
         <property name="text">
          <string> حذف همین شخص</string>
         </property>
         <property name="icon">
          <iconset resource="../booking/booking.qrc">
           <normaloff>:/choose_patient/choose.png</normaloff>:/choose_patient/choose.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>65</width>
           <height>65</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1307</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../booking/booking.qrc"/>
 </resources>
 <connections/>
</ui>
